scripts/scripts.js

```diff
@@ -166,3 +160,5 @@ async function loadPage() {
 }
 
 loadPage();
+
+document.querySelectorAll('.blog main > .section > .default-content-wrapper p').forEach((x) => { if (/^Updated/.test(x.textContent)) x.style.fontSize = '14px'; });
```

Comment:
Can we change this  so that it just adds a class "updated" to the FIRST instance of that, instead of "for each"? We will keep the font stying in the CSS.

Reply:
Sure. Please check now.

Reply:
Thanks! I think this allows the author to have some control, and also future styling for breakpoints can easily be done.